libc-test/build.rs

```diff
@@ -384,19 +384,19 @@ fn test_apple(target: &str) {
             // close calls the close_nocancel system call
             "close" => true,
 
-            // FIXME: std removed libresolv support: https://github.com/rust-lang/rust/pull/102766
+            // FIXME(ctest): std removed libresolv support: https://github.com/rust-lang/rust/pull/102766
```

Comment:
As mentioned in the other PR, preferably no `FIXME(ctest)` unless it's a bug in `ctest`. I'm not really sure what this FIXME is even for, but maybe just `FIXME(1.0)` as a general thing to look into before 1.0?

Rest lgtm.